application/templates/forms/_form_group.html

```diff
@@ -1,15 +1,18 @@
-{# Required parameters: id, legend, fields, errors, label_class, disabled #}
+{# Required parameters: id_, name, class_, fieldset_class, field_class, legend, fields, errors, label_class, disabled #}
 
 <div id="{{ id_ }}" name="{{ name }}" class="form-group {{ class_ }} {% if errors %}form-group-error{% endif %}">
   {% if fields|length == 1 %}
     {{ fields[0](disabled=disabled) }}
   {% else %}
-  <fieldset {% if id == 'areas-covered' %}class="uk-countries"{% endif %}>
+  <fieldset {% if fieldset_class %}class="{{ fieldset_class }}"{% endif %}>
     <legend>
       {{- field.label.text }} {% if errors %}<span class="error-message">{{ errors[0] }}</span>{% endif -%}
     </legend>
+    {% if field.hint or hint %}
+    <span class="form-hint">{{ field.hint|safe or hint }}</span>
```

Comment:
We should avoid using `safe` in templates, I think, and instead opt to wrap relevant pieces of text (e.g. the hint that contains markup) in `Markup()`. It feels like we don't really want anything to blanketly be rendered out raw.